src/components/ContentSidebar/ContentSidebar.js

```diff
@@ -606,15 +607,32 @@ class ContentSidebar extends PureComponent<Props, State> {
         }
     }
 
+    /**
+     * Adds a comment to the state and return the new comment.
+     *
+     * @param {Function} resolve - Resolver function for onCommentCreate() returned Promise
+     * @param {Comment} comment - The newly created comment from the API
+     * @return {void}
+     */
+    addComment(resolve: Function, comment: Comment): void {
```

Comment:
I also think this naming is a bit confusing.` onCommentCreate` makes the API call and this only updates the state. maybe follow the pattern of other success handlers, `onCommentCreateSucces`?

Reply:
done